Example/Example/ContentView.swift

```diff
@@ -0,0 +1,105 @@
+import SwiftUI
+import GEOSwift
+
+struct ContentView: View {
+    @State private var modifiableGeometry = GeometryModification()
+    @State private var tempGeometry = try! Geometry(wkt: "POLYGON((35 10, 45 45.5, 15 40, 10 20, 35 10),(20 30, 35 35, 30 20, 20 30))")
+    var body: some View {
+        VStack {
+            Text("Polygon")
+                .font(.title)
+                .padding()
+            GeometryReader { geometry in
+                Path { path in
+                    path.move(
+                        to: CGPoint(
+                            x: 0.0,
+                            y: 0.0
+                        )
+                    )
+                    modifiableGeometry.polygonModel.exterior.points.forEach { point in
+                        path.addLine(
+                            to: CGPoint(
+                                x: point.x,
+                                y: point.y
+                            )
+                        )
+                    }
```

Comment:
You should make the initial `path.move` move to the initial point of the polygon. Otherwise, you end up with an unintended line (it's not visible with the current styling, but with different styling you can see it)
![Simulator Screen Shot - iPhone 13 - 2022-08-24 at 21 43 09](https://user-images.githubusercontent.com/205658/186555240-bab2a603-c259-4eac-90cd-c15ec9fd7462.png)